### DISCUSSION TITLE
How to access a locked account

### DISCUSSION THREAD

**Discussion by AwesomePaul100** [community/community#140553] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi all,

A friend of mine kept ignoring the deadline for setting up two factor authentication on his GitHub account, and is now locked out of it. Would anyone have instructions on how to resolve this? 

Thanks,
Paul

**Comment by antidude900** [comment]
Go to [GitHub Support](https://support.github.com/contact). 
![image](https://github.com/user-attachments/assets/ad65f233-5acb-4aca-857a-5c0ba0fa9541)

Select 2FA issue and the virtual assistant there is addressing the same problem your friend is facing!
![image](https://github.com/user-attachments/assets/ca45752c-d691-4da5-8c13-e3123a0cb0f6)

**Reply by ebndev** [reply]
Hi @antidude900, thanks for taking the time to answer!

@AwesomePaul100 that response above is correct.

The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!

**Reply by AwesomePaul100** [reply]
It is not working. If we never set up a recovery code or 2fa, we cannot login. It just goes in loops.

**Comment by antidude900** [comment]
@ebndev atleast mark me as answer😅,please?

**Reply by AwesomePaul100** [reply]
Thank you. We are going to test the solution your provided now.

**Reply by AwesomePaul100** [reply]
It did not work... Without 2fa or any prior recovery keys.
